### PR TITLE
more Grants page changes

### PR DESCRIPTION
**Pull request by khangmach**
reinstate header for grants and jobs page. Hide view all jobs link on grants page

### CODE REVIEW

**Comment by khangmach** [comment]
@jamalex ready for merging and staging

**Comment by khangmach** [comment]
@or jtamiace can you review this?

File: fle_site/apps/about/templates/about/grants.html [hunk 1]
```diff
@@ -31,9 +31,9 @@
             </div> -->
 
             <div class="col-md-12 col-sm-12">
-                <div class="page-header">
+                <!--<div class="page-header">
```
Comment (jtamiace) [inline fle_site/apps/about/templates/about/grants.html]: Is there a chance that this will be added again? Better to remove commented code

Comment (khangmach) [inline fle_site/apps/about/templates/about/grants.html]: might be a chance, yeah

Comment (jtamiace) [inline fle_site/apps/about/templates/about/grants.html]: ok. passes my review then 👍

File: fle_site/static/less/greenhouse_styles.less [hunk 2]
```diff
@@ -67,7 +67,11 @@ a {
 // --------------------
 // ** GreenHouse overwrite
 // --------------------
-#header, .heading{
+#header a,
+.heading,
+.field:nth-child(7),
+.field:nth-child(8),
+.field:nth-child(9){
```
Comment (jamalex) [inline fle_site/static/less/greenhouse_styles.less]: We'll just have to remember to re-add these once we want to post actual jobs, or figure out a way to have it be selective to just the grants page.

Comment (khangmach) [inline fle_site/static/less/greenhouse_styles.less]: yep!

**Pull request merged**